src/main/java/Sb2.java

```diff
@@ -11,23 +11,40 @@
  * @author B. Clint Hall
  */
 public class Sb2 {
+    private static final String NO_JSON = "This project contains no "
```

Comment:
@whereswaldon It seems like it wants to make sure all the static methods are before any instance methods.